test/BasicAccessTest.php

```diff
@@ -56,49 +56,45 @@ public function testConstructor()
         $this->assertInstanceOf(AuthenticationInterface::class, $basicAccess);
     }
 
-    public function testIsAuthenticatedWithoutHeader()
-    {
-        $this->request
-            ->getHeader('Authorization')
-            ->willReturn([]);
-
-        $basicAccess = new BasicAccess(
-            $this->userRepository->reveal(),
-            'test',
-            $this->responseFactory
-        );
-        $this->assertNull($basicAccess->authenticate($this->request->reveal()));
-    }
```

Comment:
Just a note for anybody else reviewing: this test becomes redundant, as it is covered by the `empty` set of the `provideInvalidAuthenticationHeader` provider used by the new `testIsAuthenticatedWithInvalidData` test case.